feature-libs/product-configurator/rulebased/components/overview-attribute/configurator-overview-attribute.component.ts

```diff
@@ -8,4 +11,24 @@ import { Configurator } from '../../core/model/configurator.model';
 })
 export class ConfiguratorOverviewAttributeComponent {
   @Input() attributeOverview: Configurator.AttributeOverview;
+
+  constructor(protected breakpointService: BreakpointService) {}
+
+  extractPriceFormulaParameters(): ConfiguratorPriceComponentOptions {
+    return {
+      quantity: this.attributeOverview?.quantity,
+      price: this.attributeOverview?.valuePrice,
```

Comment:
```suggestion
      price: this.attributeOverview.valuePrice,
```
attributeOv is mandatory

Reply:
Done!